.github/workflows/test-connect.yml

```
@@ -158,3 +163,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{ fromJson(needs.set-matrix.outputs.otherDevicesMatrix) }}
+
+  all-transports:
+    needs: [build, set-matrix]
+    name: all-transports-api ${{ matrix.key }} {{ matrix.transport }}
```

```suggestion
    name: all-transports-api ${{ matrix.key }}
```
It's missing $ so it doesn't show correctly, but it can be removed altogether since it's shown in testDescription